frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.md

```diff
@@ -72,58 +72,58 @@ Letters are abcdefghijklmnopqrstuvwxyz
+    Computing Levenshtein between strings 'ut' and '' - OK
+    Computing Levenshtein between strings 'pa' and 'g' - OK
+    Computing Levenshtein between strings 'bu' and 'sx' - OK
+    Computing Levenshtein between strings 'is' and 'diy' - OK
+    Computing Levenshtein between strings 'fz' and 'unda' - OK
+    Computing Levenshtein between strings 'sem' and '' - OK
+    Computing Levenshtein between strings 'dbr' and 'o' - OK
+    Computing Levenshtein between strings 'dgj' and 'hk' - OK
+    Computing Levenshtein between strings 'ejb' and 'tfo' - OK
+    Computing Levenshtein between strings 'afa' and 'ygqo' - OK
+    Computing Levenshtein between strings 'lhcc' and '' - OK
+    Computing Levenshtein between strings 'uoiu' and 'u' - OK
+    Computing Levenshtein between strings 'tztt' and 'xo' - OK
+    Computing Levenshtein between strings 'ufsa' and 'mil' - OK
+    Computing Levenshtein between strings 'uuzl' and 'dzkr' - OK
 
 Computations in FHE
 
-    Computing Levenshtein between strings '' and '' - OK in 0.01 seconds
-    Computing Levenshtein between strings '' and 'u' - OK in 0.01 seconds
-    Computing Levenshtein between strings '' and 'nh' - OK in 0.01 seconds
-    Computing Levenshtein between strings '' and 'fmf' - OK in 0.01 seconds
-    Computing Levenshtein between strings '' and 'cljm' - OK in 0.01 seconds
-    Computing Levenshtein between strings 'v' and '' - OK in 0.01 seconds
-    Computing Levenshtein between strings 'v' and 'a' - OK in 1.75 seconds
-    Computing Levenshtein between strings 'v' and 'hp' - OK in 1.77 seconds
-    Computing Levenshtein between strings 'g' and 'ktk' - OK in 2.78 seconds
-    Computing Levenshtein between strings 'o' and 'ydqu' - OK in 3.61 seconds
-    Computing Levenshtein between strings 'ke' and '' - OK in 0.01 seconds
-    Computing Levenshtein between strings 'eu' and 'w' - OK in 1.73 seconds
-    Computing Levenshtein between strings 'hi' and 'gz' - OK in 3.53 seconds
-    Computing Levenshtein between strings 'mx' and 'tbw' - OK in 5.25 seconds
-    Computing Levenshtein between strings 'uh' and 'lgad' - OK in 7.21 seconds
-    Computing Levenshtein between strings 'xpj' and '' - OK in 0.01 seconds
-    Computing Levenshtein between strings 'cdt' and 'f' - OK in 2.53 seconds
-    Computing Levenshtein between strings 'trl' and 'rl' - OK in 5.32 seconds
-    Computing Levenshtein between strings 'zai' and 'pqo' - OK in 7.93 seconds
-    Computing Levenshtein between strings 'vac' and 'nrov' - OK in 10.73 seconds
-    Computing Levenshtein between strings 'rnay' and '' - OK in 0.01 seconds
-    Computing Levenshtein between strings 'xnfg' and 'o' - OK in 3.50 seconds
-    Computing Levenshtein between strings 'jdgl' and 'ra' - OK in 7.01 seconds
-    Computing Levenshtein between strings 'wpyq' and 'jxp' - OK in 10.67 seconds
-    Computing Levenshtein between strings 'enpt' and 'hvfb' - OK in 14.30 seconds
+    Computing Levenshtein between strings '' and '' - OK in 1.29 seconds
```

Comment:
Looks good to me, except this, this is weird :thinking:

Reply:
what is weird?

Reply:
actually, I realize that the first timing is due to the keygen. We've spent quite a lot of times on this, let me merge anyway